website/content/en/status/report-2023-04-2023-06/service-jails.adoc

```diff
@@ -1,27 +1,26 @@
-=== Service Jails - automatic jailing of rc.d services
+=== Service Jails -- automatic jailing of rc.d services
```

Comment:
While here, maybe a
switch `automatic` to a capital letter.

Reply:
If the heading must be entirely Americanised, then (from my UK perspective) I imagine: 

> `=== Service Jails -- Automatic Jailing of rc.d Services`